makelib/helm.mk

```diff
@@ -56,8 +56,10 @@ HELM_CHART_VERSION := $(VERSION:v%=%)
 # Helm Targets
 
 $(HELM_HOME): $(HELM)
+	@$(INFO) helm home and init
```

Comment:
do you think these log statements need to be committed to this upstream repo, or would it be sufficient to just add them locally within your submodule while you are debugging?

Reply:
I didn't intend to merge this PR, sorry. 
I want to test on circle-ci (because I cannot repro locally) and just opened this PR to be able to consume as a submodule (which I later realized in fact not required).